### DISCUSSION TITLE
Noob is Confused

### DISCUSSION THREAD

**Discussion by ezrakong** [rusq/slackdump#76] [Q&A] [answered]
hi y'all, total noob to git and command line but really want to use this code. I have a Mac. I downloaded the Darwin files and unpacked them, put everything in a folder named "slackdump", but when I double click the slackdump file and it opens command line,  all I see is the text below. It won't let me input any code after it runs this. What am I missing?  

> Last login: Wed Jun  8 _22:54:59 on ttys000
> You have new mail.
> /Users/macbookowner/Documents/slackdump/slackdump/slackdump ; exit;
> 
> The default interactive shell is now zsh.
> To update your account to use zsh, please run `chsh -s /bin/zsh`.
> For more details, please visit https://support.apple.com/kb/HT208050.
> MacBook-Air:~ macbookowner$ /Users/macbookowner/Documents/slackdump/slackdump/slackdump ; exit;
> Slackdump v2.0.0-rc Copyright (c) 2018-2022 rusq (build: 320a3bf)
> 
> 2022/06/09 16:06:55 no valid input and no list flags specified
> logout
> 
> Saving session...
> ...copying shared history...
> ...saving history...truncating history files...
> ...completed.
> 
> [Process completed]

**Comment by rusq** [comment, marked as answer]
Hey @ezrakong, no worries.

1. You will need to open the "Terminal" (press Cmd+Space, type "Terminal", press Return).
2. Once you see the terminal window, you need to change the directory to the folder with slackdump, i.e. if you unpacked it in "Downloads", enter the following:
```sh
cd $HOME/Downloads/slackdump
```
The shell will substitute "$HOME" with your home directory, i.e. `/Users/ezrakong/Downloads/slackdump`.

**Alternatively**, instead of 1 and 2, you can browse in Finder to the parent folder of "slackdump", i.e., if you unpacked "slackdump" in Downloads, you need to get into "Downloads" folder. Right click on "slackdump" folder and choose "Services -> New Terminal At Folder"

3. See [Usage](https://github.com/rusq/slackdump/tree/master/doc#usage) section of the Documentation, which contains the commands to dump (a) channels/users (b) generating a normal dump (c) generating dump in Slack Export format.

Hope this is helpful, any questions - let me know.

**Reply by rusq** [reply]
Also, please download the v2 - it has better compatibility with slack export viewer, if you choose to run the workspace export and view it: https://github.com/rusq/slackdump/releases/tag/v2.0.0

**Comment by ezrakong** [comment]
Thanks so much @rusq for the detailed instructions, got it up and running and its working great!

**Reply by rusq** [reply]
No problem, glad it worked for you :) I also added a section in the doc, that links to a Unix Shell Tutorial, which I think is quite well written. 
- https://github.com/rusq/slackdump/blob/master/doc/README.rst#beginner-s-guide-to-command-line
